pkg/types/builder.go

```diff
@@ -114,8 +121,15 @@ func (g *Builder) buildResource(res *schema.Resource, names ...string) (*types.N
 			req := !sch.Optional
 			comment.Required = &req
 			paramFields = append(paramFields, field)
-			g.comments.AddFieldComment(paramName, field.Name(), comment.Build())
 		}
+
+		fp := strings.Join(append(names[1:], field.Name()), ".")
```

Comment:
Actually, I think we should use `<Type Name>.<Field Name>` as the format of the key here because the same struct could be used elsewhere.

Reply:
I wouldn't prefer to include <Type Name> because it would require setting additional (already known) information in the custom configuration API.

I mean:

```
var rdsClusterCustomConfig = &resource.Configuration{
	ExternalName: resource.ExternalNameConfiguration{
		// External name config for RDS Cluster
	},
	References: map[string]resource.ReferenceConfiguration{
	--	"S3Import.BucketName": {
	--	"RDSCluster.S3Import.BucketName": {
			Type: types.PathForType(s3v1alpha1.S3Bucket{}),
		},
	},
}
```